### PR TITLE
Support for mu cfind queries

### PR DESCRIPTION
**Pull request by tbrk**
[mu](https://www.djcbsoftware.nl/code/mu/) is a tool for dealing with
e-mail messages stored in the Maildir-format, on Unix-like systems.

I didn't dare touch the autoconf files.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -113,6 +113,13 @@ m_addr_email
        (http://red.roses.de/~clemens/addressbook/) Tk program to
        search for addresses.
 
+m_mu
+       This module uses the program mu
+       (http://www.linuxstart.com/~jheinonen/abook/), a tool for
```
Comment (RolandRosenfeld) [inline README:118]: This URL redirects to a non existing host at the moment. Maybe http://www.djcbsoftware.nl/code/mu/ is a better URL here?

File: lbdbq.man.in [hunk 2]
```diff
@@ -185,6 +185,23 @@ a text based frontend to the Tk
 .BR addressbook (1)
 application.
 .TP
+.BR m_mu
```
Comment (RolandRosenfeld) [inline lbdbq.man.in]: This should be
.B m_mu

File: m_mu.sh.in [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+#     You should have received a copy of the GNU General Public License
+#     along with this program; if not, write to the Free Software Foundation,
+#     Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301,, USA.
+
+m_mu_query()
+{
+    MU=@MU@
+    MU_OPTIONS=
+
+    if [ "$MU_AFTER" -eq "$MU_AFTER" ] 2>/dev/null # test integer
```
Comment (RolandRosenfeld) [inline m_mu.sh.in]: Isn't this always true?

File: README (hunk 1 above)
Comment (tbrk) [reply README:118]: Whoops. Yes, you're right.

**Comment by tbrk** [comment]
Thank you for the merge (and for lbdb)!

**Pull request merged**